src/domain/place/utils.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Generate link to get Hsl directions
+ * @param {string} streetAddress
+ * @param {string} addressLocality
+ * @return {string}
+ */
+export const generateHslLink = (
+  streetAddress: string,
+  addressLocality: string
+) => {
+  return `http://www.reittiopas.fi/fi/?to=${streetAddress.replace(
```

Comment:
How about this HSL URL? do they support some locales like `http://www.reittiopas.fi/en/`?

Reply:
Good point. Localized hsl and service map links